.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Proposed Changes
+
+Give a brief description of the changes in this PR. If this relates to a bug or a feature request, please link it below.
+
+<!-- Link with "Closes #ISSUE-NUMBER" -->
+
+## Types of Changes
```

Comment:
Type of change or type of changes. I think there should be only one. If someone wants to do more, I would generally encourage them to create two PR's unless it is reeeeeally minor or not separable ;)